pkgs/development/python-modules/sfepy/default.nix

```diff
@@ -9,15 +9,19 @@
 , cython
 , python
 , sympy
+, meshio
+, mpi4py
+, psutil
+, openssh
 }:
 
 buildPythonPackage rec {
   name = "sfepy_${version}";
-  version = "2019.4";
+  version = "2020.4";
```

Comment:
```suggestion
  version = "2020.4";
  disabled = pythonOlderThan "3.8";
```

Reply:
Done. Using `pythonAtLeast` as I can't see where `pythonOlderThan` is  ever used in Nixpkgs.

Reply:
Its just pythonOlder. Can't remember the names.